File: DisplaBackend/DisplaBackend/Models/MovimientoCaja.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class MovimientoCaja
    {
        public int Id { get; set; }
        public DateTime Fecha { get; set; }
        public string Descripcion { get; set; }
        public decimal Monto { get; set; }
        public bool Entrada { get; set; }
        public int? IdRecibo { get; set; }
        public int? IdReciboProveedor { get; set; }
        public DateTime? FechaAnulado { get; set; }
        public bool Efectivo { get; set; }
    }
}

File: DisplaFrontend/src/app/model/movimientoCaja.ts
import { Recibo } from "./recibo";

export interface MovimientoCaja {
    Id: number,
    Fecha: Date,
    Descripcion: string,
    Monto: number,
    Entrada: boolean,
    IdRecibo: number,
    IdReciboNavigation: Recibo,
    IdReciboProveedor: number,
    // IdReciboProveedorNavigation:
    FechaAnulado: Date,
    Efectivo: boolean
}